scripts/accelerators/object.test.ts

```diff
@@ -180,4 +180,63 @@ describe("object type", () => {
 			]);
 		}
 	});
+
+	it("input merged object", () => {
+		const schema = zod.object({
```

Comment:
```ts
const schemaA = zod.object({
	test1: zod.string(),
});

const schemaB = zod.object({
	test2: zod.number(),
});

const schema = schemaA.merge(schemaB).strict();
```

This statement is a little better, it is more readable. (You would tell me on one hand, my tests are not very beautiful ^^')